### DISCUSSION TITLE
Will armv7 and Windows builds be available as release artifacts?

### DISCUSSION THREAD

**Discussion by AlphaJack** [rustic-rs/rustic#1027] [Q&A] [answered]
Hi, I noticed that the armv7 Linux and x86_64 Windows binaries available in [nightly](https://rustic.cli.rs/docs/nightly_builds.html) are both missing from the [release](https://github.com/rustic-rs/rustic/releases/tag/v0.6.1) page.

Do you have plans to make them available also as release artifacts?

**Comment by simonsan** [comment, marked as answer]
Yes, improving the release process, so it's easier to make releases more frequently, is on our TODO list. While tackling that, this problem will also be tackled, as it related to the current (temporary) release action pushing artifacts to the release.

**Comment by simonsan** [comment]
They should be available now in https://github.com/rustic-rs/rustic/discussions/1043
